README.md

```diff
@@ -10,3 +10,30 @@ CodeFlare Stack Compatibility Matrix
 | CodeFlare-SDK                | v0.4.4  |
 | InstaScale                   | v0.0.4  |
 | KubeRay                      | v0.5.0  |
+
+## Release process
+
+Prerequisite:
+- Build and release [MCAD](https://github.com/project-codeflare/multi-cluster-app-dispatcher)
+- Build and release [InstaScale](https://github.com/project-codeflare/instascale)
+- Build and release [CodeFlare-SDK](https://github.com/project-codeflare/codeflare-sdk)
+
+Release steps:
+1. Invoke [tag-and-build.yml](https://github.com/project-codeflare/codeflare-operator/actions/workflows/tag-and-build.yml) GitHub action, this action will create a repository tag, build and push operator image.
+
+2. Check result of [tag-and-build.yml](https://github.com/project-codeflare/codeflare-operator/actions/workflows/tag-and-build.yml) GitHub action, it should pass.
+
+3. Update CodeFlare Stack Compatibility Matrix in operator README.
+
+4. Create a release in CodeFlare operator repository, release notes should include new support matrix.
```

Comment:
Possible future work is adding these steps to the existing GitHub action

https://github.com/marketplace/actions/create-release

Reply:
Right